File: py/IEC61968/Metering/ServiceMultiplier.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Core.IdentifiedObject import IdentifiedObject


class ServiceMultiplier(IdentifiedObject):
    
    def __init__(self):
        super().__init__()
        self.kind = None  # Kind of multiplier.
        self.value = 1.0  # Multiplier value.

File: java/IEC61968/Metering/ServiceMultiplier.java
package IEC61968.Metering;

import IEC61970.Base.Domain.Float;
import IEC61970.Base.Core.IdentifiedObject;

/**
 * Multiplier applied at the usage point.
 * @author T. Kostic
 * @version 1.0
 * @created 25-Dec-2023 8:45:24 PM
 */
public class ServiceMultiplier extends IdentifiedObject {

	/**
	 * Kind of multiplier.
	 */
	public ServiceMultiplierKind kind;
	/**
	 * Multiplier value.
	 */
	public Float value;

	public ServiceMultiplier(){

	}
}//end ServiceMultiplier